qemu/tests/cfg/blockdev_mirror_remote_server_down.cfg

```diff
@@ -35,25 +35,25 @@
 
     # Settings for a local fs image 'data',
     # which will be exported by qemu-nbd
-    local_image_tag = data
-    image_size_data = ${image_size_data1}
-    image_name_data = data
-    image_format_data = qcow2
-    preallocated_data = falloc
-    nbd_port_data = 10810
-    nbd_export_format_data = raw
-    nbd_server_tls_creds_data = ''
-    enable_iscsi_data = no
-    enable_ceph_data = no
-    enable_gluster_data = no
-    enable_nbd_data = no
-    image_raw_device_data = no
+    local_image_tag = nbddata
```

Comment:
@aliang123 Please also update the comments. Thanks.

Reply:
Have updated it, thanks.